src/io/flutter/analytics/Analytics.java

```diff
@@ -136,6 +142,14 @@ private void sendPayload(@NotNull String hitType, @NotNull Map<String, String> a
     args.put("aiid", platformName); // Record the platform name as the application installer ID
     args.put("cd1", platformVersion); // Record the Open API version as a custom dimension
 
+    // If the Flutter SDK is provided, send the SDK version in a custom dimension.
+    if (flutterSdk != null) {
+      final FlutterSdkVersion flutterVersion = flutterSdk.getVersion();
+      if (flutterVersion.getVersionText() != null) {
+        args.put("cd2", flutterVersion.getVersionText());
```

Comment:
@jayoung-lee - just confirming that you'll be able to query the version in this format

Reply:
Perfect! Thank you.